algorithms/integration/report.py

```diff
@@ -148,7 +148,7 @@ def binned_report(binner, index, data):
 
         try:
             report["rmsd_xy"] = list(
-                indexer_sum.mean(data["xyz.rmsd"].select(data["int"]))
+                indexer_sum.mean(data["xyz.rmsd"].select(data["sum"]))
```

Comment:
Seems like a change which is a peripheral bug fix?

Reply:
i.e. if this is the case, different PR?

Reply:
It's somewhat related, it affects the output values in the table after the changes in this PR (there is a size error causing the calculation to fail and this column is populated with zero values, before the changes it happens to work ok due to the data arrays being the same size), so best to include it in here I think.